doc/md: add doc for notifications in schema monitoring (#3121)
ariga/atlas: doc/md/monitoring/07-notifications.mdx

---
title: Notifications
---

To receive notifications about schema changes in your database, you can set up notifications in Atlas Schema Monitoring.

> Notifications can be sent to Slack, Workplace, or a plain webhook URL.

Reviewer: ```suggestion
Notifications can be sent to Slack, Workplace, or any HTTP endpoint.
```

> 
> 

Reviewer: ```suggestion

```
Remove double lines in the rest of the doc as well please.

#### Setting up Notifications

> To set up notifications, go to the **webhooks** tab of the database you want to monitor and click on the **Add Webhook** button.

Reviewer: 1. Log in to your Atlas Cloud account at https://auth.atlasgo.cloud.
2. Click on "Monitoring" in the top-level navigation on the left side of the screen. 
3. Choose ..

Choose where you want to send the notification, and fill in the required information.

<details>
<summary>Screenshot Example</summary>

![](https://atlasgo.io/uploads/cloud/monitoring/webhook-create.png)
</details>

> You can send ping webhooks to test the connection and verify that the notification is working as expected by clicking on the **Send Test Notification** button.

Reviewer: To test your configuration, use the ...

<details>
<summary>Screenshot Example</summary>

![](https://atlasgo.io/uploads/cloud/monitoring/send-ping.png)
</details>

After setting up the notification, you will receive notifications whenever there are a schema changes in the database.

> Lest create new table in our database and see how the notification works.

Reviewer: Lest -> Lets

This is a new section. Needs missing subheader

---

> title: Notifications

Reviewer: are we calling this "webhooks" or "notifications" or "subscriptions"? we need to be consistent in the UI, docs, and marketing

Author: I think it is best to go with webhooks, I will change it to be consistent

---

To receive notifications about schema changes in your database, you can set up notifications in Atlas Schema Monitoring.
Notifications can be sent to Slack, Workplace, or any HTTP endpoint.

#### Setting up Notifications

1. Log in to your Atlas Cloud account at https://auth.atlasgo.cloud.

2. Click on **Monitoring** in the top-level navigation on the left side of the screen.

3. Choose the database you want to monitor and click on the **Webhooks** tab.

4. Click on the **Add Webhook** button and fill in the required information.

> <details>

Reviewer: 5. Select the integration type

Add a tabs component for each type documenting the fields in the form and their meaning.
